galite-demo/galite-vaadin/src/main/kotlin/org/kopi/galite/demo/database/Migration.kt

```diff
@@ -333,20 +332,20 @@ fun addTaxRule(id: Int, taxName: String, rate: Int, information: String? = null)
 }
 
 fun addBills() {
-  addBill(0, "Bill address 0", LocalDate.parse("2018-09-13"), BigDecimal("3129.7"), 0, ExposedBlob(byteArrayOf(244.toByte(), 244.toByte(), 244.toByte())))
-  addBill(1, "Bill address 1", LocalDate.parse("2020-02-16"), BigDecimal("1149.24"), 1, ExposedBlob(byteArrayOf(244.toByte(), 244.toByte(), 244.toByte())))
-  addBill(2, "Bill address 2", LocalDate.parse("2019-05-13"), BigDecimal("219.6"), 2, ExposedBlob(byteArrayOf(244.toByte(), 244.toByte(), 244.toByte())))
-  addBill(3, "Bill address 3", LocalDate.parse("2019-01-12"), BigDecimal("146.9"), 3, ExposedBlob(byteArrayOf(244.toByte(), 244.toByte(), 244.toByte())))
+  addBill(0, "Bill address 0", LocalDate.parse("2018-09-13"), BigDecimal("3129.7"), 0, 7652806)
+  addBill(1, "Bill address 1", LocalDate.parse("2020-02-16"), BigDecimal("1149.24"), 1, 2662474)
+  addBill(2, "Bill address 2", LocalDate.parse("2019-05-13"), BigDecimal("219.6"), 2, 3502417)
+  addBill(3, "Bill address 3", LocalDate.parse("2019-01-12"), BigDecimal("146.9"), 3, 5281517)
```

Comment:
Since this is only to test the color, I thin it's better to use colors available as constants : 
`addBill(0, "Bill address 0", LocalDate.parse("2018-09-13"), BigDecimal("3129.7"), 0, java.awt.Color.LIGHT_GRAY.rgb)`
`addBill(1, "Bill address 1", LocalDate.parse("2020-02-16"), BigDecimal("1149.24"), 1, java.awt.Color.MAGENTA.rgb)`
`addBill(2, "Bill address 2", LocalDate.parse("2019-05-13"), BigDecimal("219.6"), 2, java.awt.Color.ORANGE.rgb)`
`addBill(3, "Bill address 3", LocalDate.parse("2019-01-12"), BigDecimal("146.9"), 3, java.awt.Color.PINK.rgb)`